examples/plotting/wcsaxes_plotting_example.py

```diff
@@ -0,0 +1,64 @@
+"""
+=====================================
+Plotting points on a Map with WCSAxes
+=====================================
+
+This example demonstrates the plotting of point, line and arch in pixel coordinate,
+world coordinate, and SkyCoords respectively when plotting a map with WCSAxes.
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.data.sample
+import sunpy.map
+from sunpy.coordinates.utils import GreatArc
+
+###############################################################################
+# Firstly, We will start by creating a Map using an AIA 171 image.
+# Secondly, Let's first plot a line on the map by using coordinates in arcseconds.
+# The array below `xx` and `yy` are the x and y coordinates that define a from
+# the Sun center (at 0, 0) to the point (500, 500) in arcsecs.
+# The standard plotting in WCSAxes expects pixel coordinates, but we can plot
+# world coordinates (i.e. arcsec) by using the `transform` keyword.
+# Note that the coordinates need to be in degrees rather than arcseconds.
+
+my_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+fig = plt.figure(figsize=(8, 8))
+ax = fig.add_subplot(projection=my_map)
+my_map.plot(axes=ax, clip_interval=(1, 99.9)*u.percent)
+
+xx = np.arange(0, 500)
+yy = xx
```

Comment:
Do we need to define this?

Reply:
Yes I guess. As we are plotting a line from center of the image to 500 arcsec and xx and yy coordinates are same.